README.md

```diff
@@ -20,4 +20,32 @@ This repository is configured for Nat Pryce's [_adr-tools_](https://github.com/n
 
 API definitions are in OAS (Swagger 2.0).
 
-# The Digital Citizenship Initiative
+## Deploy Azure PaaS
```

Comment:
I would say perhaps, using Terraform terminology, `Configuring the Azure infrastructure`

Reply:
ok